code/game/gamemodes/miniantags/borer/borer.dm

```diff
@@ -564,7 +565,8 @@
 	host.reset_perspective(null)
 	host.machine = null
 
-	var/mob/living/H = host
+	var/mob/living/carbon/H = host
```

Comment:
Borer code is unspeakably bad...it's like the creator couldn't decide if you could only infest carbons, all living mobs, or only humans.

Reply:
Greetings from the future, where I am scraping for borer fixes in a rework because when I wrote them I couldn't decide if you could only infest carbons, all living mobs, or only humans.